.github/workflows/pre-merge_actions.yml

```diff
@@ -76,7 +75,7 @@ jobs:
             )
           )
           }}
-        uses: jaumann/github-bumpversion-action@master
```

Comment:
There is not an official `bumpversion` action from GitHub, no? As long as everything works the same it really does not matter. It would only be nice to check if we rely on any feature (e.g. `env.DEFAULT_BUMP`) which may specifically rely on the subscribed action. 

I do however like that it is now tired to a version and not just something like `master` which might change over time 👍🏻

Reply:
It's essentially the same action, but fixed by another developer.